### PR TITLE
Fri 35/add pmt to tk config imgspc

### PR DESCRIPTION
**Pull request by baptisteguidoux**
New `site-packages` dir to distribute dependencies.
`Third Party Notices.md` added
`pmt` and `appdirs` added as dependency

Before any app launch, the `site-packages` dir is added to `PYTHONPATH`

### CODE REVIEW

File: Third Party Notices.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+This package contains third-party software components governed by the license(s) indicated below:
```
Comment (baptisteguidoux) [inline Third Party Notices.md:1]: This is new

File: env/includes/settings/tk-multi-launchapp.yml [hunk 2]
```diff
@@ -20,6 +20,7 @@ includes:
 settings.tk-multi-launchapp:
   use_software_entity: true
   hook_before_register_command: "{config}/tk-multi-launchapp/before_register_command.py"
+  hook_before_app_launch: "{config}/tk-multi-launchapp/before_app_launch.py"
```
Comment (baptisteguidoux) [inline env/includes/settings/tk-multi-launchapp.yml:23]: The new hook configured

File: hooks/tk-multi-launchapp/before_app_launch.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2013 Shotgun Software Inc.
```
Comment (baptisteguidoux) [inline hooks/tk-multi-launchapp/before_app_launch.py:1]: The new hook

File: Third Party Notices.md (hunk 1 above)
Comment (lassond) [reply Third Party Notices.md:1]: Beautiful, thanks!

**Review by lassond — APPROVED** [review]

**Pull request merged**